hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/rollback/BaseRollbackHelper.java

```diff
@@ -125,12 +129,17 @@ List<Pair<String, HoodieRollbackStat>> maybeDeleteAndCollectStats(HoodieEngineCo
         final StoragePath filePath;
         try {
           String fileId = rollbackRequest.getFileId();
+          HoodieTableVersion tableVersion = metaClient.getTableConfig().getTableVersion();
 
           writer = HoodieLogFormat.newWriterBuilder()
               .onParentPath(FSUtils.constructAbsolutePath(metaClient.getBasePath(), rollbackRequest.getPartitionPath()))
               .withFileId(fileId)
-              .withDeltaCommit(instantToRollback.getTimestamp())
+              .withLogWriteToken(CommonClientUtils.generateWriteToken(taskContextSupplier))
```

Comment:
The write token is meaningless because the rollback tasks have different task context with the writer task.

Reply:
I'll check this out.. I was trying to avoid doing FS listings or sth expensive, without this being passed in

Reply:
at-least for Spark - this makes sense I think. each task is a rollback to a given file group.. So they'd have different write tokens. My intention don't want to compute this from storage ..  lmk if you have concerns with proceeding